bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Region.java

```diff
@@ -588,36 +598,44 @@ public void translate (Point pt) {
 	storeAndApplyOperationForAllHandles(operation);
 }
 
-private long getHandleForInitialZoom() {
-	return win32_getHandle(this, initialZoom);
-}
-
 private void storeAndApplyOperationForAllHandles(Operation operation) {
 	operations.add(operation);
-	zoomToHandle.forEach((zoom, handle) -> operation.apply(handle, zoom));
+	zoomToHandle.forEach((zoom, handle) -> operation.apply(handle));
+}
+
+private <T> T applyUsingAnyHandle(Function<RegionHandle, T> function) {
+	if (zoomToHandle.isEmpty()) {
+		return applyUsingTemporaryHandle(device.getDeviceZoom(), operations, function);
+	} else {
+		return function.apply(zoomToHandle.values().iterator().next());
+	}
```

Comment:
NIT: move outside of `else`
```suggestion
	return function.apply(zoomToHandle.values().iterator().next());

```

Reply:
Adapted in a01f1a88baaf56c2c7c80094bd49c13e5193c655